source/core/Events.cpp

```diff
@@ -1011,6 +1011,9 @@ VIREO_FUNCTION_SIGNATURE3(IsNERefnum, RefNumVal, RefNumVal, Boolean);
 //------------------------------------------------------------
 DEFINE_VIREO_BEGIN(Events)
 
+    // EventSpec used for event structure configuration
+    DEFINE_VIREO_TYPE(EventSpec, "c(e(UInt32 eventSource)e(UInt32 eventType)e(UInt32 controlUID)e(UInt32 dynIndex)");
```

Comment:
Is it OK to have spaces in between the different "e()"s? That would make it a bit easier for me to read.

Reply:
Sure.